slow_odgi/chop_by.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import mygfa
+import random
+
+
+def print_n(graph):
+    """Prints out a reasonable parameter to pass to `chop`.
+    Specifically: a random number between 2 and the length of the longest seq.
+    """
+    # random.seed(4)
+    longest_seq = max([len(s.seq) for s in graph.segments.values()])
+    print (random.randint(2,longest_seq))
+
+if __name__ == "__main__":
+    graph = mygfa.Graph.parse(sys.stdin)
+    print_n(graph)
+
```

Comment:
This is an interesting strategy! I'm guessing it didn't suffice to hard-code a number that is shorter than most longest seqs, like 4 or something? I just ask because this does add another source of complexity; if the simpler approach will work, maybe we should consider it.

Reply:
I used to have the hard-coded style (I just used 3) but I developed this little script as part of my fancy new "better testing" push. If 3 is just as good, I'm happy to do that!